src/agent/README.md

```diff
@@ -49,6 +49,11 @@ $ rustup target add "${arch}-unknown-linux-musl"
 $ sudo ln -s /usr/bin/g++ /bin/musl-g++
 ```
 
+ppc64le-only: Manually install protoc, e.g.
+```bash
+$ sudo dnf install protobuf-compiler
+```
```

Comment:
I'm fine about this change here as it's, but I'd also suggest to hack https://github.com/kata-containers/tests/blob/777d62e8a51d7bad6c0ddd98002fd67a71ef20d2/.ci/install_kata_image.sh#L26-L57, check for the arch, and then install protobuf-compiler in case it's ppc64le.

Does this sound reasonable?

Reply:
Hi @clnperez, sorry for piggyback -- can we add s390x as well? It also needs protoc additionally.

Reply:
@fidencio i can look into that. my first thought/concern is that this could get tricky b/c package names and installers would vary from distro to distro. is there something in the scripts that installs a package in this way (outside the docker way)?

Reply:
@clnperez, don't make my comment a blocker, as we don't have the CI running por PPC right now.

Let's focus on fixing the CI related issues / improve CI deployment whenever we have a PPC machine to plug into the CI. :-)

Reply:
@fidencio thanks. <3 that's what we're working on internally. if we can get all the tiny kinks out of all the build bits, we'll be in good shape when that hardware is finally set up